main.py

```diff
@@ -41,44 +45,87 @@ def create_backend(bucket: str, key: str, region: str = 'us-east-2'):
     f.write('terraform {\n')
     f.write('  backend "s3" {\n')
     f.write('    bucket = "' + bucket + '"\n')
-    f.write('    key    = "' + key + '.json"\n')
+    f.write('    key    = "' + key + '"\n')
```

Comment:
Just FYI you can write multiline strings using:
```
f.write('''
terraform {\n
  backend "s3" {\n
  ...
''')
```

Reply:
i couldn't get variable injection to work in the multiline string for lines 47-49